themes/docs-new/layouts/shortcodes/system_requirements_server_hardware.md

```diff
@@ -9,29 +9,23 @@ preferred.
 
 **All Deployments**
 
--   64-bit architecture
+- 64-bit CPU architecture
+- CPU support for SSE4.2 extensions (Xeons starting in 2007 and Opterons in 2012)
 
 **Standalone Deployments**
 
--   4 total cores (physical or virtual)
--   8 GB of RAM or more
--   5 GB of free disk space in `/opt`
--   10 GB of free disk space in `/var`
+- 4 total cores (physical or virtual)
+- 8 GB of RAM or more
+- 5 GB of free disk space in `/opt`
+- 10 GB of free disk space in `/var`
 
 <div class="admonition-note">
 
 <p class="admonition-note-title">Note</p>
 
 <div class="admonition-note-text">
 
-The RAM requirement can be lowered down to a minimum of 4 GB of RAM if
-the number of Chef Infra Client runs (CCRs) per minute are low (i.e.
-less than 33 CCRs/min). See [Capacity
-Planning](/server_overview.html#capacity-planning) for more information
-on how this metric affects scalability.
-
-
-
+The RAM requirement can be lowered down to a minimum of 4 GB of RAM if the number of Chef Infra Client runs (CCRs) per minute are low (i.e. less than 33 CCRs/min). See [Capacity Planning](/server_overview.html#capacity-planning) for more information on how this metric affects scalability.
```

Comment:
strike 'i.e.' and use (less than 33 CCRs/min)

Reply:
Updated